src/imitation/algorithms/dagger.py

```diff
@@ -496,6 +531,30 @@ def extend_and_update(
         logging.info(f"New round number is {self.round_num}")
         return self.round_num
 
+    def _get_trainable_predict_fn(
```

Comment:
Similar to with BC -- I'm concerned about special-casing this feature into the algorithm.

What about if a year later we want to add some other component of the observation that is to be presented to humans but ignored by the policy? Do we need to go through every algorithm and update them to include this as well?